### PR TITLE
Remove dates and unused terms in mapper

### PR DESCRIPTION
**Pull request by rosibaj**
https://kidsfirst.atlassian.net/wiki/spaces/KFDRC/pages/713359389/ETL+Updates+June+2018

- removing extraneous dates and foreign keys that slipped in
- HPO extra fields removed and observed renames, but structure remains the same

### CODE REVIEW

File: es_mapper.yaml [hunk 1]
```diff
@@ -104,65 +112,64 @@ target-mappings:
...
+                          - observed
                           - kf_id
                           properties:
                             ancestral_hpo_ids:
                               type: keyword
-                            negative_hpo_ids:
+                            phenotype_not_observed:
                               type: keyword
-                            hpo_ids:
+                            phenotype_observed:
```
Comment (junjun-zhang) [inline es_mapper.yaml]: Does `phenotype_observed` refer to `hpo_id_phenotype` field in data service? If that's the case, should it be better to name the field as `hpo_phenotype_observed`? Particularly consider that there is another phenotype field: `snomed_id_phenotype` (possibly new ones later).

Same for `phenotype_not_observed` field, changing it to `hpo_phenotype_not_observed`?

Comment (rosibaj) [reply es_mapper.yaml]: @junjun-zhang Addresses `snomed_phenotype_observed` and `snomed_phenotype_not_observed`

File: es_mapper.yaml [hunk 2]
```diff
@@ -104,80 +112,91 @@ target-mappings:
...
-              - snomed_id
+              - observed
               - kf_id
               properties:
-                phenotype:
-                  type: text
                 ancestral_hpo_ids:
                   type: keyword
-                negative_hpo_ids:
+                phenotype_not_observed:
```
Comment (junjun-zhang) [inline es_mapper.yaml]: `phenotype_not_observed` -> `hpo_phenotype_not_observed`

File: es_mapper.yaml [hunk 3]
```diff
@@ -104,80 +112,91 @@ target-mappings:
...
                   type: keyword
-                hpo_ids:
+                phenotype_observed:
```
Comment (junjun-zhang) [inline es_mapper.yaml]: `phenotype_observed` -> `hpo_phenotype_observed`

File: es_mapper.yaml [hunk 4]
```diff
@@ -250,85 +281,95 @@ target-mappings:
...
                               properties:
-                                phenotype:
-                                  type: text
                                 ancestral_hpo_ids:
                                   type: keyword
-                                negative_hpo_ids:
+                                phenotype_not_observed:
                                   type: keyword
-                                hpo_ids:
+                                phenotype_observed:
```
Comment (junjun-zhang) [inline es_mapper.yaml]: `phenotype_observed` -> `hpo_phenotype_observed`

File: es_mapper.yaml [hunk 5]
```diff
@@ -250,85 +281,95 @@ target-mappings:
...
-                              - snomed_id
+                              - observed
                               - kf_id
                               properties:
-                                phenotype:
-                                  type: text
                                 ancestral_hpo_ids:
                                   type: keyword
-                                negative_hpo_ids:
+                                phenotype_not_observed:
```
Comment (junjun-zhang) [inline es_mapper.yaml]: `phenotype_not_observed` -> `hpo_phenotype_not_observed`

File: es_mapper.yaml [hunk 6]
```diff
@@ -250,85 +281,95 @@ target-mappings:
...
+                  - snomed_id_phenotype
                   - participant_id
-                  - snomed_id
+                  - observed
                   - kf_id
                   properties:
                     ancestral_hpo_ids:
                       type: keyword
-                    negative_hpo_ids:
+                    phenotype_not_observed:
```
Comment (junjun-zhang) [inline es_mapper.yaml]: `phenotype_not_observed` -> `hpo_phenotype_not_observed`

File: es_mapper.yaml [hunk 7]
```diff
@@ -250,85 +281,95 @@ target-mappings:
...
                       type: keyword
-                    hpo_ids:
+                    phenotype_observed:
```
Comment (junjun-zhang) [inline es_mapper.yaml]: `phenotype_observed` -> `hpo_phenotype_observed`

**Review by junjun-zhang — COMMENTED** [review]
There are a few places missed for `phenotype_observed` and `phenotype_not_observed`

**Review by junjun-zhang — APPROVED** [review]
Looks good

**Pull request merged**